sklearn/compose/tests/test_target.py

```diff
@@ -299,6 +299,12 @@ def test_transform_target_regressor_count_fit(check_inverse):
 
 
 class DummyRegressorWithExtraFitParams(DummyRegressor):
+    def request_check_input(self, *, fit=None):
```

Comment:
Is the SLEP 006 defined two new methods request methods: `request_sample_weight` and `request_groups` or is it opening it to any `request_*` method?

Reply:
Open to any `request_*` method. The point is to have an explicit API for exactly those keys that the consumer knows how to handle.